pallets/anchors/src/migration.rs

```diff
@@ -1,81 +0,0 @@
-use super::*;
-
-pub mod fix_evict_date {
-	use frame_support::{log, traits::Get, weights::Weight};
-
-	use super::*;
-
-	pub const HARDCODED_EVICTED_DATE: u32 = 19200;
-
-	#[cfg(feature = "try-runtime")]
-	use frame_support::ensure; // Not in prelude for try-runtime
-
-	#[cfg(feature = "try-runtime")]
-	pub fn pre_migrate<T: Config>() -> Result<(), &'static str> {
-		ensure!(
-			LatestEvictedDate::<T>::get() == None,
-			"State already initialized"
-		);
-		Ok(())
-	}
-
-	pub fn migrate<T: Config>() -> Weight {
-		if LatestEvictedDate::<T>::get().is_none() {
-			LatestEvictedDate::<T>::put(HARDCODED_EVICTED_DATE);
```

Comment:
I can see this value in Altair, so the migration seems to be correctly applied. It can be removed 👍🏻